### PR TITLE
Moving sequence variation methods from PsmWriter to mzLib

### PR DESCRIPTION
**Pull request by acesnik**

### CODE REVIEW

**Comment by acesnik** [comment]
Ready for review @rmillikin @rmmiller22

File: Proteomics/ProteolyticDigestion/PeptideWithSetModifications.cs [hunk 1]
```diff
@@ -411,6 +411,71 @@ public PeptideWithSetModifications Localize(int j, double massToLocalize)
             return peptideWithLocalizedMass;
         }
 
+        /// <summary>
+        /// Determines whether a peptide includes a splice site
+        /// </summary>
+        /// <param name="pep"></param>
+        /// <param name="site"></param>
+        /// <returns></returns>
+        public bool Includes(SpliceSite site)
```
[the remark this answers is not in the document]
Comment (acesnik) [reply Proteomics/ProteolyticDigestion/PeptideWithSetModifications.cs]: Will do, but if you can only put in a splice site, is it really that vague? 😅

**Comment by acesnik** [comment]
100% diff hit by tests. Duly ready for review, @rmillikin @rmmiller22 :)

**Comment by rmmiller22** [comment]
@rmillikin do you want to review for approval as well or should I merge this in?

**Comment by rmillikin** [comment]
nope, unless you want me to review, though I'm much less familiar with this than you two

**Comment by rmmiller22** [comment]
Nope. I just didn't want to step on toes. I will go ahead and merge this in.

**Pull request merged**